extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -604,24 +609,35 @@ export class PublishDatabaseDialog {
 		const baseImages = getDockerBaseImages(this.project.getProjectTargetVersion());
 		const baseImagesValues: azdataType.CategoryValue[] = baseImages.map(x => { return { name: x.name, displayName: x.displayName }; });
 
-		// add preview string for 2022
-		// TODO: remove after 2022 is GA
-		if (this.project.getProjectTargetVersion() === constants.targetPlatformToVersion.get(SqlTargetPlatform.sqlServer2022)) {
-			const sqlServerImageIndex = baseImagesValues.findIndex(image => image.displayName === constants.SqlServerDockerImageName);
-			if (sqlServerImageIndex >= 0) {
-				baseImagesValues[sqlServerImageIndex].displayName = constants.SqlServerDocker2022ImageName;
-			}
-		}
-
 		this.baseDockerImageDropDown = view.modelBuilder.dropDown().withProps({
 			values: baseImagesValues,
 			ariaLabel: constants.baseDockerImage(name),
 			width: cssStyles.publishDialogTextboxWidth,
 			enabled: true
 		}).component();
 
+		const imageInfo = baseImages.find(x => x.displayName === (<azdataType.CategoryValue>this.baseDockerImageDropDown?.value)?.displayName);
+		const imageTags = await uiUtils.getImageTags(imageInfo!, this.project.getProjectTargetVersion(), true);
+
+		this.imageTagDropDown = view.modelBuilder.dropDown().withProps({
+			values: imageTags,
+			value: imageTags[0],
+			ariaLabel: constants.imageTag,
+			width: cssStyles.publishDialogTextboxWidth,
+			enabled: true,
+			editable: true,
```

Comment:
Do we want this to be editable?

Reply:
yup, that's how filtering gets enabled. We do the same thing for the database dropdowns in this extension